packages/cli/test/fixtures/openapi/3.0/petstore-expanded.yaml

```diff
@@ -0,0 +1,156 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Swagger Petstore
```

Comment:
Do we really need both JSON and YAML version of petstore spec? If you want to test handling of both JSON and YAML inputs, then I am proposing to keep only one format in git and generate the other format on the fly inside the test.

Reply:
Please note they are two different specs:
- petstore-expanded.json is for swagger 2.0 
- petstore-expanded.yaml is for openapi 3.0

Reply:
I see. Can you make that difference more obvious please, e.g. by encoding it in the file name?

- petstore-expanded-swagger.json
- petstore-expanded-openapi.json

Reply:
> Can you make that difference more obvious please, e.g. by encoding it in the file name

@raymondfeng ☝️

Reply:
Fixed